src/validator/CreditCardChecking.java

```diff
@@ -0,0 +1,5 @@
+package validator;
+
+public interface CreditCardChecking {
```

Comment:
The name does not follow the class convention. It should not be CreditCardChecking, but CreditCardChecker or CreditCardValidator (we already have that class). In general, this interface is implemented only by CreditCardValidator. So in my opinion, we can remove this interface, because we don't expect a second implementation.